dev/source/docs/mavlink-requesting-data.rst

```diff
@@ -146,6 +146,37 @@ Send a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`_
 - param2: depends on message requested; see that message's definition for details.
 - param3 to param7: 0 (not used)
 
+Specifying Message Rates in a File
+----------------------------------
+
+At boot ArduPilot will populate the initial message intervals from files found in either ROMFS or in the filesystem.
+
+On ChibiOS-based boards the SD card will be searched for specially-named files in the APM subdirectory.
+
+Configuration files can be included in ROMFS (i.e. compiled into the image) by specifying their path in the relevant board's hwdef file.
+
+.. code-block::
+
+   ROMFS message-intervals-chan0.txt message-intervals-chan0.txt
```

Comment:
so, the first param is the file name that will be added into ROMFS and must be "message-intervals-chanX.txt" format for each channel, and the second param is the path to a text file with the date to be used in that file?   having the same name without some more explanation (assuming I have figured it out correctly) is confusing..
perhaps ROMFS message-intervals-chan0.txt <path to text file with data to load into the ROMFS file> ?

Reply:
also, need to add at very top as 5th method to set rates....but I can add after merge if you want